app/views/list/remove-modal.html

```diff
@@ -0,0 +1,24 @@
+<div class="modal-header">
+	<h4 class="modal-title">
+		Delete list
+	</h4>
+</div>
+
+<div class="modal-body">
+
+	Are you sure you want to delete list <strong>{{list.name}}</strong>?
+
+</div>
+
+<div class="modal-footer">
+
+	<button class="btn btn-danger" type="button" ng-click="remove()">
+		<i class="fa fa-exclamation-triangle"></i>
```

Comment:
Exclamation mark has to emphasize destructive result of clicking that button. As I wanted to do it, 'yes' meaning was just not right for this context, so I decided to use 'delete' instead. That in turn caused 'no' to be replaced with 'cancel'